dev/Interop/StoragePickers/PickerLocalization.cpp

```diff
@@ -9,14 +9,47 @@
 #include "PickerLocalization.h"
 #include <winrt\base.h>
 #include <winrt\Microsoft.Windows.ApplicationModel.Resources.h>
-#include <iostream>
+#include <array>
+
+
+const winrt::hstring priFileName = L"Microsoft.WindowsAppRuntime.pri";
+winrt::hstring GetPriFilePath()
+{
+    HMODULE hModule = nullptr;
+    // use GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT, the module handle does not need to be released
+    // GetPriFilePath should only be called once to initialize the static priFilePath variable, thus thread safe
+    if (GetModuleHandleExW(GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS | GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT,
+        reinterpret_cast<LPCWSTR>(&GetPriFilePath), &hModule))
+    {
+        std::array<wchar_t, MAX_PATH + 1> buffer;
+        auto pathLength = GetModuleFileName(hModule, buffer.data(), MAX_PATH + 1);
+        if (pathLength > 0)
+        {
+            auto dllPathString = std::wstring(buffer.data(), pathLength);
+            std::filesystem::path dllPath(dllPathString);
+            if (!std::filesystem::is_regular_file(dllPath))
+            {
+                return priFileName;
+            }
+            std::filesystem::path parentDir = dllPath.parent_path();
+            auto priFilePath = parentDir / std::wstring{ priFileName };
+            if (std::filesystem::exists(priFilePath))
+            {
+                return winrt::hstring{ priFilePath.wstring() };
+            }
+        }
+    }
+    return priFileName;
+}
 
 namespace PickerLocalization {
-    const winrt::hstring priPath = L"Microsoft.WindowsAppRuntime.pri";
+
     winrt::hstring GetStoragePickersLocalizationText(winrt::hstring key)
     {
+        static winrt::hstring priPath = GetPriFilePath();
```

Comment:
NIT: `static winrt::hstring priPath{ GetPriFilePath() };`

[ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

same comment applies throughout